test/expected/timestamp.out

```diff
@@ -509,50 +509,137 @@ FROM unnest(ARRAY[
  Sun Nov 05 03:05:00 2017 EST | Sun Nov 05 02:00:00 2017
 (4 rows)
 
-SELECT time, time_bucket(10, time)
+SELECT time, time_bucket(10::smallint, time), time_bucket(10::int, time), time_bucket(10::bigint, time)
```

Comment:
can you add `AS time_bucket_bigint` and so forth to make the columns in the output easier to interpret.

Reply:
done